pkg/tiles/tile.go

```diff
@@ -0,0 +1,85 @@
+package tiles
+
+import (
+	buildings "github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/buildings"
+	connectionMod "github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/connection"
+	featureMod "github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/feature"
+)
+
+/*
+Immutable object
+*/
+type Tile struct {
+	Features  []featureMod.Feature
+	HasShield bool
+	Building  buildings.Bulding
+}
+
+func (tile *Tile) Cities() [][]connectionMod.Side {
+	var cities [][]connectionMod.Side
+	for _, feature := range tile.Features {
+		if feature.FeatureType == featureMod.City {
+			cities = append(cities, feature.Connections)
+		}
+	}
+	return cities
+}
+
+func (tile *Tile) CitiesAppendConnection(connections []connectionMod.Side) {
```

Comment:
If the API was accepting `Feature`, we could merge `*AppendConnection()` methods into single `AppendFeature()`.

Reply:
as you wish, removed

Reply:
I guess you went for having the user use `append(tile.Features, ...)` rather than having a separate method, seems fine.